worker.go

```diff
@@ -233,20 +237,23 @@ func (wm WorkerMetrics) appendExportedMetric(res []*metricpb.Metric, exp metricE
 }
 
 // NewWorker creates, and returns a new Worker object.
-func NewWorker(id int, cl *trace.Client, logger *logrus.Logger, stats scopedstatsd.Client) *Worker {
+func NewWorker(id int, runningInGlobalVeneur bool, cl *trace.Client, logger *logrus.Logger, stats scopedstatsd.Client) *Worker {
```

Comment:
This is called `runningInGlobalVeneur`, but you're passing in `ret.IsLocal()`, which seems like a bug?

Separately, I'd prefer to store it as `isLocal`, since it's more consistent with the naming scheme we use for the method on `Server`. Though that may require flipping all the values (or not, depending on if this is a bug or not).